REFERENCE.md

```diff
@@ -0,0 +1,77 @@
+# Reference
+<!-- DO NOT EDIT: This document was generated by Puppet Strings -->
+
+## Table of Contents
+
+**Classes**
+
+_Public Classes_
+
+* [`lldpd`](#lldpd): A module to manage lldpd and related facts
+
+_Private Classes_
+
+* `lldpd::install`: Manages the lldpd package
+* `lldpd::repo`: Manages the yum or apt repositories
+* `lldpd::service`: Manages the lldpd service + facts
+
+## Classes
+
+### lldpd
+
+lldpd
+
+#### Examples
+
+##### Basic usage
+
+```puppet
+include lldpd
+```
+
+#### Parameters
+
+The following parameters are available in the `lldpd` class.
+
+##### `manage_facts`
+
+Data type: `Boolean`
+
+Enable or disable the fact deployment. Doesn't work on FreeBSD yet
+
+##### `manage_service`
+
+Data type: `Boolean`
+
+Enable or disable the service management
+
+##### `manage_repo`
+
+Data type: `Boolean`
+
+Enable or disable the repository setup
+
+##### `manage_jq`
+
+Data type: `Boolean`
+
+Enable or disable the installation of jq (doesn't manage maybe needed third-part repositories like EPEL)
```

Comment:
`maybe needed` isn't that clear. `optional` isn't that great either.
`third-part` -> `third-party`

Reply:
ugh awful phrasing. I fixed it.